components/x-eventpromo/src/Link.jsx

```diff
@@ -0,0 +1,9 @@
+import { h } from '@financial-times/x-engine';
+
+const Link = ({url, attrs = {}, children}) => {
+	return (
+		<a href={url} {...attrs}>{children}</a>
```

Comment:
Does this need to be a HOC (could components implement the `<a>` anchor element themselves?)